platform/android/java/lib/src/org/godotengine/godot/io/file/DataAccess.kt

```diff
@@ -135,6 +135,15 @@ internal abstract class DataAccess(private val filePath: String) {
 		seek(positionFromBeginning)
 	}
 
+	fun resize(length: Long): Boolean {
```

Comment:
Following on @mhilbrunner suggestion, let's return an int to indicate other errors besides just `FAILED`.

Based on the [documentation](https://developer.android.com/reference/java/nio/channels/FileChannel#truncate(long)) the following exceptions are possible:

- NonWritableChannelException / ClosedChannelException -> `ERR_FILE_CANT_OPEN`
- IllegalArgumentException -> `ERR_INVALID_PARAMETER`
- Anything else -> `FAILED`